projects/schematics/src/add/index.ts

```diff
@@ -48,13 +48,36 @@ function setProjectSettings(options: NgAddOptions) {
 }
 
 // Checks if a version of Angular is compatible with current or next
-function getVersion(ngVersion: string, clrVersion: string): string {
+export function getVersion(ngVersion: string, clrVersion: string): string {
+  const majorNgVersion = Number.parseInt(ngVersion.split('.')[0].replace(/\D/g, ''));
+
+  if (majorNgVersion >= 12) {
+    return getVersionsAfterTwelve(ngVersion, clrVersion);
+  } else {
+    // Otherwise, just link to installed version for latest or next releases
+    return getVersionsBeforeTwelve(ngVersion, clrVersion);
+  }
+}
+
+function getVersionsAfterTwelve(ngVersion: string, clrVersion: string): string {
```

Comment:
This really just needs to return the Angular major version number and return it, doesn't need to calculate a diff.